bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -2314,4 +2316,45 @@ private function addTemplatingEngineNode(ArrayNodeDefinition $rootNode): void
             ->end()
         ->end();
     }
+
+    private function addWriteTargetNodes(ArrayNodeDefinition $rootNode): void
+    {
+        $storageNode = $rootNode
+            ->children()
+            ->arrayNode('storage')
+            ->addDefaultsIfNotSet()
+            ->children();
+
+        $this->addStorageNode($storageNode, 'image_thumbnails');
+        $this->addStorageNode($storageNode, 'custom_reports');
+        $this->addStorageNode($storageNode, 'video_thumbnails');
+        $this->addStorageNode($storageNode, 'document_types');
+        $this->addStorageNode($storageNode, 'web_to_print');
+        $this->addStorageNode($storageNode, 'predefined_properties');
+        $this->addStorageNode($storageNode, 'predefined_asset_metadata');
+        $this->addStorageNode($storageNode, 'staticroutes');
+        $this->addStorageNode($storageNode, 'perspectives');
+        $this->addStorageNode($storageNode, 'custom_views');
+        $this->addStorageNode($storageNode, 'data_hub');
+        $this->addStorageNode($storageNode, 'object_custom_layouts');
+
+    }
+
+    private function addStorageNode(NodeBuilder $node, string $name): void
+    {
+        $node->
+            arrayNode($name)
+                ->addDefaultsIfNotSet()
+                ->children()
+                    ->variableNode('target')
```

Comment:
I think we should have default values here, then we don't need to set defaults in the  `LocationAwareConfigRepository`, we just rely on this config then. 
The pathes should use `%kernel.project_dir%`